src/api-reference/invoice/purchase-request.markdown

```diff
@@ -5,21 +5,23 @@ layout: reference
 
 # Purchase Request
 
-## Get Started
+### Overview
+<p><strong>Important</strong>: This API is only available for direct integrations with an existing SAP Concur client. If you are a Partner looking to build an App Center App using this API, please reach out to your SAP Concur Representative.</p>
```

Comment:
This information belongs in the Limitations section, please move it there.

Reply:
Fixed in 6f49cd1b

Reply:
@sarraloew  Please check now